mavflightview: add GUID Plotting

jmachuca77 opened the pull request:
Add plotting of GUID log points when they are locations.
![Screen Shot 2021-11-11 at 8 48 39 PM](https://user-images.githubusercontent.com/1664405/141405892-22f87622-090a-4575-a021-d96c533df70b.png)

jmachuca77 commented on MAVProxy/tools/mavflightview.py:
```diff
@@ -104,7 +104,6 @@ def create_imagefile(options, filename, latlon, ground_width, path_objs, mission
                 (mavutil.mode_mapping_rover.values(),colour_map_rover),
                 (mavutil.mode_mapping_tracker.values(),colour_map_tracker),
                 (mavutil.mode_mapping_sub.values(),colour_map_submarine),
-                (mavutil.mode_mapping_blimp.values(),colour_map_blimp),
```
@tridge This line was giving me an error when trying to show the map, I branched from master, so this issue is also present on master. 
```
() $ MAVExplorer.py 2021-11-03\ 14-52-01.bin 
MAV> map
MAV> ERROR in command []: 'module' object has no attribute 'mode_mapping_blimp'
```

peterbarker replied:
@jmachuca77 you need to update pymavlink

tridge reviewed (changes requested):
happy to accept this change if you remove the change to blimp

jmachuca77 commented:
@tridge @peterbarker I removed the blimp change and updated my pymavlink... but im still getting the same error. If this PR works for you then please merge it. Example log file containing GUID points attached to test if needed:
[2021-11-03 14-52-01.bin.zip](https://github.com/ArduPilot/MAVProxy/files/7529262/2021-11-03.14-52-01.bin.zip)
.

jmachuca77 commented:
I managed to fix my pymavlink installation and tested this and its working correctly for me now. So should be good to merge.
